store/dynamodb/dynamodb.go

```diff
@@ -415,6 +415,7 @@ func (ddb *DynamoDB) AtomicDelete(key string, previous *store.KVPair) (bool, err
 	}
 
 	expAttr := make(map[string]*dynamodb.AttributeValue)
+	// FIXME previous.LastIndex may lead to a nil pointer dereference
```

Comment:
It's easily fixable now I think, we could just wrap that single statement with a:

```go
if previous != nil {
    [...]
}
```

As explained in another comment that's a larger issue with this method signature and the use of a pointer as an optional parameter, but we'll have to live with it for now.